gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/service/ServiceDistributionPluginTests.groovy

```diff
@@ -500,7 +500,7 @@ class ServiceDistributionPluginTests extends GradleTestSpec {
                 .find({ it.name.endsWith("-manifest-classpath-0.0.1.jar") })
         classpathJar.exists()
         readFromZip(classpathJar, "META-INF/MANIFEST.MF")
-                .contains('Class-Path: guava-19.0.jar produces-manifest-') // etc
+                .contains('Class-Path: guava-19.0.jar root-project-manifest-') // etc
```

Comment:
- the previous version of nebula-test this repo was on didn't generate maven repo when using DependencyGraph with the version of gradle it was on. Therefore, I bumped nebula-test to latest, but the latest version of nebula-test used the crazy temporary folder annotation this projected used before. I switched to junit's TemporaryFolder, but it generates the folder as a UUID, so i had to set rootProject.name in settings.gradle so set it to somethign sane (root-project) and hence this chage...
phew